packages/root-cms/ui/components/DocEditor/fields/ImageField.tsx

```diff
@@ -8,13 +8,20 @@ import {joinClassNames} from '../../../utils/classes.js';
 import {uploadFileToGCS} from '../../../utils/gcs.js';
 import {FieldProps} from './FieldProps.js';
 
-export const IMAGE_MIMETYPES = [
-  'image/png',
+/** Mimetypes accepted by the image input field. */
+const IMAGE_MIMETYPES = [
+  'image/gif',
   'image/jpeg',
+  'image/png',
   'image/svg+xml',
   'image/webp',
 ];
 
+/** Builds the `accept` attribute for the `input` field.*/
+export function buildAcceptAttribute(exts?: string[]) {
```

Comment:
Remove the "export" since I think this is only used in this file.

Reply:
Done